bspacks/esy.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "get-js-of-ocaml",
+  "notes": "This is to get js_of_ocaml",
+  "private": true,
+  "version": "3.3.3",
+  "dependencies": {
+    "@opam/js_of_ocaml": "*",
+    "@opam/menhir": " >= 20170418.0.0  <= 20171013.0.0",
+    "@opam/ocaml-migrate-parsetree": "*"
+  },
+  "peerDependencies": {
+    "ocaml": "4.2.3004"
+  },
+  "devDependencies": {
+    "ocaml": "~4.2.3004"
+  },
+  "esy": {
+    "build": [
+      ["echo", "No-op"]
+    ],
+    "_dependenciesForNewEsyInstaller": {
```

Comment:
This shouldn't be needed. Was it? It was just a hack for some special cased packages to make sure some old projects will continue to install correctly. It no longer applies.